scene/resources/animation.h

```diff
@@ -65,28 +65,19 @@ class Animation : public Resource {
 
 private:
 	struct Track {
-		TrackType type;
-		InterpolationType interpolation;
-		bool loop_wrap;
+		TrackType type = TrackType::TYPE_ANIMATION;
+		InterpolationType interpolation = INTERPOLATION_LINEAR;
+		bool loop_wrap = true;
 		NodePath path; // path to something
-		bool imported;
-		bool enabled;
-		Track() {
-			interpolation = INTERPOLATION_LINEAR;
-			imported = false;
-			loop_wrap = true;
-			enabled = true;
-		}
+		bool imported = false;
+		bool enabled = false;
```

Comment:
I missed this on review, but this changed the default value from `true` to `false`.

Reply:
Fixed by #46585.